### ISSUE TITLE
postgres-exporter error

### ISSUE THREAD

**Issue by nhudson** [timescale/helm-charts#425]
When you enable postgres-exporter it will start showing this error in the logs. 

```
ts=2022-08-22T08:36:52.303Z caller=collector.go:196 level=error msg="collector failed" name=bgwriter duration_seconds=25837.993198715 err="sql: Scan error on column index 2, name \"checkpoint_write_time\": conv │
│ erting driver.Value type float64 (\"3.23725841e+08\") to a int: invalid syntax"
```

**Comment by nhudson** [comment]
Testing this locally, it would seem that it's working as expected.

**Comment by nhudson** [comment]
Looks like it's an actual bug with postgres_exporter https://github.com/prometheus-community/postgres_exporter/pull/666

This is fixed with version `0.11.1` which was released a few days ago.  So this is already fixed in this helm-chart and in tobs.
